1. Building a To-Do List Without Framework/submission/index.js

```diff
@@ -104,22 +105,28 @@ const saveTodo = (e) => {
 };
 
 const clearTodo = () => {
-    const clearInfo = `
+        if (todoList.length){
+            const approveClear = confirm("Are you sure you want to delete your all items");
+            if(approveClear){
+                `${localStorage.removeItem("todoList")}
+                ${(todoList = [])}
+                ${renderList()}`
+                const clearInfo = `
                 <div class="text-center alert alert-primary alert-dismissible fade show shadow border-0" role="alert"  data-aos="zoom-in-down">
-                <strong>${
-                    todoList.length > 0
-                        ? "Your items have been successfully removed."
-                        : "Please add any item."
-                }
+                <strong>Your items have been successfully removed.</strong>
                     <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
-                </div>`;
+                </div>`;            
     $("#save-info").html(clearInfo);
-    todoList.length > 0 // if array length is null then do not show confirm msg.
-        ? `${confirm("Are you sure you want to delete your all items")}
-                ${localStorage.removeItem("todoList")}
-                ${(todoList = [])} // array made empty
-                ${renderList()}`
-        : "";
```

Comment:
You should not do this. This is not at all readable and may not get properly formatted by eslint or other formatters.

Use a simple `if` condition.